src/microbe_stage/ProcessSystem.cs

```diff
@@ -134,6 +139,12 @@ public ProcessSystem(Node worldRoot)
         result.Osmoregulation = Constants.ATP_COST_FOR_OSMOREGULATION * hexCount *
             membrane.OsmoregulationFactor;
 
+        // World settings may be null if this is the first generation
```

Comment:
Would it be possible to fix the code structure so that the process system is not given invalid data? I'd much prefer a "proper" design where this workaround here could be avoided.